### DISCUSSION TITLE
Geo LOCK

### DISCUSSION THREAD

**Discussion by IVEN2024** [ant-media/Ant-Media-Server#6653] [Ideas]
when you all add GEO LOCK to stream ?

**Comment by burak-58** [comment]
Hi @IVEN2024,
When we get high demand for a feature, we get that feature to the plan. 
Also if there is a good business opportunity, we may prioritize the feature request. Sometimes AMS users want us to add a feature by covering the cost of the implementation and according to availibility we may prioritize it.
We haven't got a high demand for ths feature. If you think there is an opportunity to implement this feature, please let us know.

**Comment by lastpeony** [comment]
I think this would be a cool feature :)

As a workaround webrtc play requests can be geo locked.
Follow this documentation to enable webhook play auth with ip address
https://antmedia.io/docs/guides/stream-security/webhook-stream-authorization/

You can check users ip address on your application server and find its location. Based on this you can reject the play.
